common/alias.i

```diff
@@ -1,3 +1,7 @@
+UPBTN       = $0800
+DNBTN      = $0400
```

Comment:
You can define these in your input module instead of the alias.i -- If you want to keep it localized to your changes.

```suggestion
DNBTN       = $0400
```